src/datepicker/datepicker.ts

```diff
@@ -83,47 +94,48 @@ export interface NgbDatepickerNavigateEvent {
       </div>
     </ng-template>
 
-    <div class="ngb-dp-header">
-      <ngb-datepicker-navigation *ngIf="navigation !== 'none'"
-        [date]="model.firstDate"
-        [months]="model.months"
-        [disabled]="model.disabled"
-        [showSelect]="model.navigation === 'select'"
-        [prevDisabled]="model.prevDisabled"
-        [nextDisabled]="model.nextDisabled"
-        [selectBoxes]="model.selectBoxes"
-        (navigate)="onNavigateEvent($event)"
-        (select)="onNavigateDateSelect($event)">
-      </ngb-datepicker-navigation>
-    </div>
-
-    <div #months class="ngb-dp-months" (keydown)="onKeyDown($event)">
-      <ng-template ngFor let-month [ngForOf]="model.months" let-i="index">
-        <div class="ngb-dp-month">
-          <div *ngIf="navigation === 'none' || (displayMonths > 1 && navigation === 'select')"
-                class="ngb-dp-month-name">
-            {{ i18n.getMonthFullName(month.number, month.year) }} {{ i18n.getYearNumerals(month.year) }}
+    <ng-template #defaultContentTemplate>
+      <div class="ngb-dp-header">
+        <ngb-datepicker-navigation *ngIf="navigation !== 'none'"
+          [date]="model.firstDate"
+          [months]="model.months"
+          [disabled]="model.disabled"
+          [showSelect]="model.navigation === 'select'"
+          [prevDisabled]="model.prevDisabled"
+          [nextDisabled]="model.nextDisabled"
+          [selectBoxes]="model.selectBoxes"
+          (navigate)="onNavigateEvent($event)"
+          (select)="onNavigateDateSelect($event)">
+        </ngb-datepicker-navigation>
+      </div>
+
+      <div #months class="ngb-dp-months" (keydown)="onKeyDown($event)">
```

Comment:
- keyboard handling should be moved to the `datepikcerMonthView` as well I guess
- also `#months` is not used anymore